### DISCUSSION TITLE
File Uploading with node.js and vercel

### DISCUSSION THREAD

**Discussion by dev-zohaibamin** [vercel/community#2450] [Help] [answered]
### Summary

My application is build with node.js and deployed on vercel  i have a images folder in my app directory its locally uploading image sucessfully but on server side mean on vercel its not storing the images in that folder

**Comment by leerob** [comment, marked as answer]
These two examples might help:

- https://github.com/vercel/examples/tree/main/storage/blob-starter
- https://github.com/vercel/examples/tree/main/solutions/aws-s3-image-upload
